api/swagger_v2.yaml

```diff
@@ -12,6 +12,10 @@ info:
     communicating with components via Redfish.
     It also allows administrators to create partitions and groups for other uses.
 
+    ## Deprecation Notice: V1 of the HMS service has been deprecated as of CSM version 0.9.3. 
+    The V1 HSM API’s will be removed in the CSM version 1.4 release. 
+    All consumers of the V1 HSM API interface will need to move to the V2 interface prior to the CSM 1.4 release.
+
```

Comment:
Do we need to add back in the swagger_v1.yaml file for the v1 API?

Reply:
The v1 APIs are getting removed later. I think keeping the v1 swagger doc around would only serve to enable development against them.